src/state/DB.js

```diff
@@ -167,6 +167,14 @@ class DB {
         );
     }
 
+	 getMainSheet(info) {
```

Comment:
can you get the indentation to match

Reply:
It lines up for me

Reply:
i think you're using tabs while it uses spaces or vice versa